lib/assessment.md

```diff
@@ -41,7 +56,9 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+    all elements that belong to a class big =
+    ID content pane = div
+    important / red =
```

Comment:
I think you've misunderstood what this question is asking. You were directed to give the _CSS selector_ which referred to the element or group of elements described.